### PR TITLE
add cci_lccs harvesting

### PR DESCRIPTION
**Pull request by zacharlie**

### CODE REVIEW

**Comment by alexgleith** [comment]
Ready for you to review and check out, @zacharlie

File: deafrica/data/copernicus_cci.py [hunk 1]
```diff
@@ -62,7 +49,8 @@ def get_version_from_year(year: str) -> str:
     """
     try:
         year_num = int(year)
-        if year_num not in range(1992, 2020):
+        # Only allow from 1992 to 2020, inclusive
+        if year_num not in range(1992, 2021):
```
Comment (flippant-heron) [inline deafrica/data/copernicus_cci.py:53]: The source https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-land-cover currently only lists (supports?) dates up to 2020. Minor issue though, as I believe the available date range should change in a month or so anyway.

**Review by zacharlie — COMMENTED** [review]
LGTM! Thanks!

File: deafrica/data/copernicus_cci.py (hunk 1 above)
Comment (alexgleith) [reply deafrica/data/copernicus_cci.py:53]: range(1992,2021) finishes on 2020 anyway...

**Review by alexgleith — APPROVED** [review]

**Comment by alexgleith** [comment]
Next is merging this, creating a release, and then running in DEV using an Argo template

**Pull request merged**